upstart_msh/cob-start

```diff
@@ -92,9 +77,28 @@ export ROS_IP=`rosrun robot_upstart getifip eth0`
 export ROS_MASTER_URI=http://$ROS_IP:11311
 export ROS_LOG_DIR=/tmp
 rosrun robot_upstart mklaunch $JOB_FOLDER robot:=$ROBOT > $LAUNCH_FILENAME
-su msh -c "sh /etc/ros/indigo/cob.d/robot_launch.sh" 
-PID=$!
 
-#log info "robot.launch: Started roslaunch as background process, PID $PID"
-echo "$PID" > /tmp/cob-start.pid
-#wait "$PID"
+#start roscore inside tmux sessiond2
+su $USER -l -c "tmux new -d -s roscore"
+su $USER -l -c "tmux send -t roscore.0 'sh /etc/ros/mydistro/cob.d/roscore_start.sh' ENTER"
+
+#wait for roscore
+until rostopic list; do sleep 1; done
+
+rm -rf /tmp/runningVMS
+su mimic -c "ssh myrobotname-h1 'vboxmanage list runningvms'" > /tmp/runningVMS
+
+if grep -q myrobotname-win "/tmp/runningVMS"; then
+  echo "VM already running"
+else
+  echo "starting VM..."
+  su mimic -c "ssh myrobotname-h1 'export DISPLAY=:0 && virtualbox --startvm myrobotname-win --fullscreen'"&
+fi
+
+
+#start bringup inside tmux session
+su $USER -l -c "tmux new -d -s bringup"
+su $USER -l -c "tmux send -t bringup 'sh /etc/ros/mydistro/cob.d/robot_launch.sh' ENTER"
+
+ssh $USER@myrobotname-t1 "tmux new -d -s statemachine"
```

Comment:
needs to be started on t2! see https://github.com/ipa320/msh/blob/indigo_dev/msh_bringup/launch/application.xml#L97